README.md

```diff
@@ -0,0 +1,256 @@
+# py2rb.py
```

Comment:
Removing README.rst and creating README.md causes `pip install 'git+https://github.com/nanobowers/py2rb@new_versions'` to fail because setup.py requires `README.rst`.